### ISSUE TITLE
Plugin does not report various issues

### ISSUE THREAD

**Issue by chrisgrieser** [ismay/stylelint-no-unsupported-browser-features#228]
Hi, thanks for the great plugin, it is pretty much what I have been looking for!

I tried it out and it seems some issues are not properly recognized by the plugin. So I have for example this css file, and I am targeting Chrome Version 80 and upwards. 

```css
body:has(a) {
  hyphens: auto;
  overflow-y: clip;
}
```

This means that all three lines should be reported, since `:has`, `hyphens`, and `overflow-y: clip` are all not supported. To be sure, I checked that [can-i-use](https://caniuse.com/?search=overflow-y%3A%20clip), too.

However, stylelint only reports `hyphens` and ignoring the other two. I also tried upgrading all my npm packages, but it seems not to make a difference. Here are my version numbers and my the relevant part of my stylelint config:

```text
├── postcss@8.4.5
├── stylelint-no-unsupported-browser-features@5.0.2
└── stylelint@14.2.0
```

```json
"plugins": [
  "stylelint-no-unsupported-browser-features"
],

"plugin/no-unsupported-browser-features": [true, {
  "browsers": ["Chrome > 80"],
  "ignore": ["css-masks"]
}],
```

**Comment by ismay** [comment]
I should add this to the issue template or readme, but basically this plugin just wraps [doiuse](https://github.com/anandthakker/doiuse). Anything that has to do with the detecting of errors should be resolved in that lib.

**Comment by chrisgrieser** [comment]
I see, thanks!
